### PR TITLE
Update centralized-deployment-of-add-ins.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a line stating that the module works only with Basic auth. As there is ambiguity on whether this works with Modern auth with MFA disabled.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4685

File: microsoft-365/admin/manage/centralized-deployment-of-add-ins.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ When the tool finishes running, it produces an output file in comma-separated (.
 - Supported Mailbox - If they are on an OAuth-enabled mailbox
 
 > [!NOTE]
-> Multifactor authentication is not supported when using the Central Deployment PowerShell module.
+> Multifactor authentication is not supported when using the Central Deployment PowerShell module. The module only works with basic authentication.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/centralized-deployment-of-add-ins.md]:
```suggestion
> Multifactor authentication is not supported when using the Central Deployment PowerShell module. The module only works with Basic authentication.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@kwekuako Gentle reminder

**Comment by yogkumgit** [comment]
@efrene  Please help as it is from addins area.

**Pull request merged**